content/2016.06.kinto-admin-1.0.md

```diff
@@ -0,0 +1,48 @@
+Title: Kinto-admin reaches 1.0
+Date: 2016-06-10
+Lang: en
+Url: kinto-admin-1.0
+Slug: kinto-admin-1.0
+Summary: Introducing kinto-admin, a Web UI for managing your Kinto data
+
+When choosing a database, it's a good idea to study a few important criteria: how nice is the API? Is it well documented? Will it scale? What is the tooling available? Tooling is especially important as it's what will make your life as a developer easier. Basically [kinto-admin] wants to be to Kinto what [phpMyAdmin] is to MySQL. We've mentioned kinto-admin a few times on this blog already, and today we're announcing the release of [version 1.0].
```

Comment:
I think there's a disconnect here between "it will make your life as a developer easier" and "kinto-admin wants to be to Kinto what phpMyAdmin is to MySQL". The first part sounds like general introductory material, but then by the end of the paragraph you say you've already mentioned kinto-admin a few times. If you've covered it before, maybe you can take out the introductory stuff and just put links, something like "We've mentioned kinto-admin a few times on this blog already, but to summarize, it wants to be for Kinto what phpMyAdmin is for MySQL." Or if you want to keep the introductory material, then maybe insert something after it that connects to the next part: "While developing Kinto, we've paid special attention to the tooling, in particular kinto-admin, which wants to be for Kinto what phpMyAdmin is to MySQL".